import-service/serverless.yml

```diff
@@ -0,0 +1,52 @@
+service: import-service
+frameworkVersion: "3"
+
+plugins:
+  - serverless-webpack
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: us-east-1
+  httpApi:
+    cors: true
+  environment:
+    FILE_UPLOAD_BUCKET_NAME: ${self:custom.fileBucketName}
+  iam:
+    role:
+      statements:
+        - Effect: "Allow"
+          Action: "s3:*"
+          Resource: "arn:aws:s3:::${self:custom.fileBucketName}"
+
+        - Effect: "Allow"
```

Comment:
Looks like a duplication of previous statement on line 19

Reply:
above is for the bucket and the below one is for files inside the bucket. Do we need to do that or allowing bucket automatically allow to all objects inside bucket?

Reply:
you are right, I missed difference in last line. We need both.
For the line 20 as value I used s3:ListBucket, not just *.